### ISSUE TITLE
Allow changing the text of the "afbreken" button. 

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#4984]
### Thema / Theme

Form designer

### Omschrijving / Description

requested by: Ven 29, odru 62

As a form designer i want to change the "Afbreken" button text as "Afbreken" is not B1 dutch, and additionally some forms might benefit from a custom text for this button which cannot be set like the other form buttons currently can.

**Comment by sergei-maertens** [comment]
Refinement: please suggest a button text that is B1 (Taiga suggests "annuleren") and we'll change it to that. Adding code to manage button translations for a vague future potential is not the best use of our resources:

> ...additionally some forms might benefit...

too vague and I couldn't find any concrete use cases in the mentioned Taiga issues :)
